debezium-server/debezium-server-core/pom.xml

```diff
@@ -132,9 +141,50 @@
                     <enableAssertions>true</enableAssertions>
                     <systemProperties>
                         <test.type>IT</test.type>
+                        <test.debezium.apicurio.profile.is.active>false</test.debezium.apicurio.profile.is.active>
                     </systemProperties>
                 </configuration>
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>apicurio-ext-json</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <skipTests>${skipITs}</skipTests>
+                            <enableAssertions>true</enableAssertions>
+                            <systemPropertyVariables>
+                                <test.debezium.converter.format>json</test.debezium.converter.format>
+                                <test.debezium.apicurio.profile.is.active>true</test.debezium.apicurio.profile.is.active>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>apicurio-avro</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <skipTests>${skipITs}</skipTests>
+                            <enableAssertions>true</enableAssertions>
+                            <systemPropertyVariables>
+                                <test.debezium.converter.format>avro</test.debezium.converter.format>
+                                <test.debezium.apicurio.profile.is.active>true</test.debezium.apicurio.profile.is.active>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
```

Comment:
There should be also `assembly` profile that will execute both it tests, one by one. Please see https://github.com/debezium/debezium/blob/master/debezium-connector-mysql/pom.xml#L453 for example how this can be achieved.